packages/firestore/test/unit/specs/existence_filter_spec.test.ts

```diff
@@ -286,4 +246,347 @@ describeSpec('Existence Filters:', [], () => {
       );
     }
   );
+
+  /**
+   * Test existence filter with bloom filter.
+   */
+  specTest(
+    'Full re-query is skipped when bloom filter can identify documents deleted',
+    [],
+    () => {
+      const query1 = query('collection');
+      const docA = doc('collection/a', 1000, { v: 1 });
+      const docB = doc('collection/b', 1000, { v: 2 });
+      const bloomFilterProto = generateBloomFilterProto({
```

Comment:
I think we should make `numOfBits` and `hashCount` optional when calling `generateBloomFilterProto()` and instead just pick some reasonable values if they are not specified. For example, numOfBits=8000 and hashCount=6 will probably yield zero false positives ever in these tests.

Then, tests like this where these two parameters are irrelevant will be more readable because this extra "noise" about the numOfBits and hashCount can be removed.

What do you think?

Reply:
sounds good. I will set the numOfBits to a less ambitious number, like 100 instead of 800.